swagger.json

```diff
@@ -959,6 +959,182 @@
+                            "$ref": "#/definitions/ErrorResponse"
+                        }
+                    }
+                }
+            }
+        },
+        "/fleet/unassigned_driving_segments/annotate": {
+            "patch": {
+                "tags": [
+                    "Fleet", "Default"
+                ],
+                "summary": "/fleet/unassigned_driving_segments/annotate",
+                "description": "Annotate an unassigned driving segment.",
+                "operationId": "annotate_unassigned_driving_segment",
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/accessTokenParam"
+                    },
+                    {
+                        "name": "annotateUnassignedDrivingSegmentParams",
+                        "required": true,
+                        "in": "body",
+                        "schema": {
+                            "type": "object",
+                            "required": [
+                                "vehicleId",
+                                "startMs",
+                                "annotation",
+                            ],
+                            "properties": {
+                                "vehicleId": {
+                                    "type": "integer",
+                                    "format": "int64",
+                                    "description": "Vehicle ID for the segment.",
+                                    "example": 212014918086169
+                                },
+                                "startMs": {
+                                    "type": "integer",
+                                    "format": "int64",
+                                    "description": "Milliseconds UNIX time when this segment started.",
+                                    "example": 1523059200000
+                                },
+                                "annotation": {
+                                    "type": "string",
+                                    "description": "Annotation to give this segment.",
+                                    "example": "Drive time"
+                                }
+                            }
+                        }
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "The unassigned driving segment that was annotated.",
+                        "schema": {
+                            "$ref": "#/definitions/UnassignedDrivingSegment"
+                        }
+                    },
+                    "default": {
+                        "description": "Unexpected error.",
+                        "schema": {
+                            "$ref": "#/definitions/ErrorResponse"
+                        }
+                    }
+                }
+            }
+        },
+        "/fleet/unassigned_driving_segments/assign": {
```

Comment:
Same as above, could we change this to `PATCH /fleet/unassigned_driving_segments/:id`?

Reply:
So would that make these the same endpoints? Or do you mean
`/fleet/unassigned_driving_segments/:id/assign`

Reply:
I was thinking we `PATCH /fleet/unassigned_driving_segments/:id`, and the `body` JSON includes the new `pending_driver_id` value.